io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
licenses?

Reply:
Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/6/commits/0385a6d8936ff3b7c647d680cf0b9a29f3ebc140